### DISCUSSION TITLE
Processes API showing up empty

### DISCUSSION THREAD

**Discussion by piattnd** [dotnet/dotnet-monitor#5603] [Q&A] [answered]
Hello folks.

I came across a few other threads where other people had similar issues.  One of them came down to a configuration issue (on the .sock variable) and the other seemed to be traced back to an async operation in the application that was somehow causing DotNetMonitor to drop the process.

The application I am attempting to monitor is an async process running in .net core 3.1.  I have run the example deployment yaml file provided with this set of documents and everything works fine.  When I try to replicate the exact same setup for my application though, I get the behavior above (/processes API returns an empty array).

I have tried increasing the log level on dotnet monitor, but I have so far failed to get any higher log level than informational.  Has anyone run into similar behavior and what steps did you take to narrow down the issue?

**Comment by jander-msft** [comment]
The issue is likely due to .NET Core 3.1 not supporting the reversed server connection mode (what we call "listen" mode for .NET Monitor). This may be the same issue as #1552

**Comment by piattnd** [comment, marked as answer]
Thanks for the tip.  I took a look at that link and I tried implementing both suggestions there (yours to remove the environment variables and mount to /tmp and the other individual who indicated changing from Listen to Connect).

When I removed the environment variables, the info API returned DiagnosticPortName was null (probably expected) and the logs endpoint returns `{"status":400,"detail":"Unable to discover a target process."}` for both.

I may just try to expedite upgrading the project from .net core 3.1.  Appreciate the quick response!
